cli/bash/commands/get-one-identity.sh

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) 2019, Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -e
+
+if [ ! -z "$DEBUG" ]; then
+    set -x
+fi
+
+. "$PEP_CLI_DIR/common.sh"
+
+cli_help_get_one_identity() {
+  echo "
+Usage: pep get-one-identity [<options>]
+
+Options:
+  -a <ip_or_dns>            pelion cloud api url (default: 'https://api.us-east-1.mbedcloud.com')
```

Comment:
You could move the readonly variables before this function and reference this URL from them so that this documented URL doesn't become out of date.

Reply:
Sure, will do that.